dask_sphinx_theme/__init__.py

```diff
@@ -1,6 +1,7 @@
 from os import path
+from . import _version
 
-__version__ = "2.0.3"
+__version__ = _version.get_versions()["version"]
```

Comment:
I see `dask` and `distributed` also define a `__git_revision__` 

https://github.com/dask/dask/blob/aad132a5dad414d987c6358536f41ebf946094b1/dask/__init__.py#L10

I'm not sure exactly where that's used, so I'm not sure if it makes sense to add that attribute here or remove it from `dask` / `distributed`

Reply:
Looks like `__git_revision__` is just a nice-to-have which `dask`, `pandas`, and a few other libraries add. I don't think it's a big deal here, but could be added is someone asks for it